python/README.md

```diff
@@ -16,72 +32,68 @@ poetry install
 poetry shell
 ```
 
-Make sure you have an
-[Open AI API Key](https://openai.com/api/) or
-[Azure Open AI service key](https://learn.microsoft.com/azure/cognitive-services/openai/quickstart?pivots=rest-api)
 
-Copy those keys into a `.env` file in this repo (see the `.env.example` file):
+# Get Started with Semantic Kernel ⚡
 
-```
-OPENAI_API_KEY=""
-OPENAI_ORG_ID=""
-AZURE_OPENAI_API_KEY=""
-AZURE_OPENAI_ENDPOINT=""
-```
-
-### Quickstart ⚡
+## Example: Running a simple prompt.
 
 ```python
 import semantic_kernel as sk
-import semantic_kernel.ai.open_ai as sk_oai
+from semantic_kernel.ai.open_ai import OpenAITextCompletion
 
 kernel = sk.create_kernel()
 
-# This requires a `.env` file in your current
-# directory (see above)
+# Prepare OpenAI backend using credentials stored in the `.env` file
 api_key, org_id = sk.openai_settings_from_dot_env()
+kernel.config.add_text_backend("dv", OpenAITextCompletion("text-davinci-003", api_key, org_id))
 
-kernel.config.add_text_backend(
-    "davinci-002", sk_oai.OpenAITextCompletion(
-        "text-davinci-002", api_key, org_id
-    )
-)
-
-sk_prompt = """
-{{$input}}
+# Wrap your prompt in a function
+prompt = kernel.create_semantic_function("""
+1) A robot may not injure a human being or, through inaction,
+allow a human being to come to harm.
 
-Give me the TLDR in exactly 5 words.
-"""
+2) A robot must obey orders given it by human beings except where
+such orders would conflict with the First Law.
 
-text_to_summarize = """
-    1) A robot may not injure a human being or, through inaction,
-    allow a human being to come to harm.
+3) A robot must protect its own existence as long as such protection
+does not conflict with the First or Second Law.
 
-    2) A robot must obey orders given it by human beings except where
-    such orders would conflict with the First Law.
+Give me the TLDR in exactly 5 words.""")
 
-    3) A robot must protect its own existence as long as such protection
-    does not conflict with the First or Second Law.
-"""
-
-tldr_function = kernel.create_semantic_function(
-    sk_prompt,
-    max_tokens=200,
-    temperature=0,
-    top_p=0.5,
-)
+# Run your prompt
+print(prompt()) # => Robots must not harm humans.
+```
 
-summary = await kernel.run_on_str_async(text_to_summarize, tldr_function)
-output = str(summary.variables).strip()
-print("Output: " + output)
+## Example: Turn prompts into **reusable functions** with input parameters.
 
-# Output: Robots must not harm humans.
+```python
```

Comment:
For an example to put at the front in the Python README, I think it would be good to show both a Semantic Function and Native function being used together by the Semantic Kernel. I got a lot of feedback from some hackathon people wondering how this works.

Reply:
the intention is to differentiate prompts from functions, starting simple and then showing how to reuse prompts. The first example addresses the basic question: how to run a prompt.  The concept of functions is not obvious, and the second examples shows how to move from prompts to semantic functions, reinforcing the message about reusability using the same prompt multiple times. There's been a lot of feedback and confusion about SK being too complex to run a prompt and unclear what functions are.